middleware/csrf/custom_errorfunc/main.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022 CloudWeGo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/hertz-contrib/csrf"
+	"github.com/hertz-contrib/sessions"
+	"github.com/hertz-contrib/sessions/cookie"
+)
+
+func myErrFunc(_ context.Context, ctx *app.RequestContext) {
+	if ctx.Errors.Last() == nil {
+		err := fmt.Errorf("myErrFunc called when no error occurs")
+		ctx.String(400, err.Error())
```

Comment:
This seems a little bit confusing. Why return 400 when there is no error happen in business logic?
Maybe add some comments to clarify what this example wants to express.